### PR TITLE
Implement config_content parameter

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -88,6 +95,8 @@
   Optional[Stdlib::Filesource] $config_file_v6  = undef,
   Optional[String[1]] $config_template_v6       = undef,
   Boolean $manage_repo                          = false,
+  Optional[String[1]] $config_content_v4        = undef,
```
Comment (traylenator) [inline manifests/init.pp]: New variables so new tests?

Comment (bastelfreak) [reply manifests/init.pp]: added some

File: spec/classes/bird_spec.rb [hunk 2]
```diff
@@ -198,6 +205,78 @@
           it { is_expected.to contain_yumrepo('bird') }
           it { is_expected.to contain_package('bird') }
         end
+        context 'with config_file_v4 and config_template_v4' do
```
Comment (traylenator) [inline spec/classes/bird_spec.rb]: What about testing the positive case were the _content parameters are specified.

**Comment by traylenator** [comment]
I missed that the new parameters should be added to README.md given that the other are listed there.

**Pull request merged**